src/components/index.js

```diff
@@ -3,6 +3,7 @@ export {default as WebexActivity} from './WebexActivity/WebexActivity';
 export {default as WebexActivityStream} from './WebexActivityStream/WebexActivityStream';
 export {default as WebexDataProvider, AdapterContext} from './WebexDataProvider/WebexDataProvider';
 export {default as WebexInterstitialMeeting} from './WebexInterstitialMeeting/WebexInterstitialMeeting';
+export {default as WebexInMeeting} from './WebexInMeeting/WebexInMeeting';
```

Comment:
If we're importing alphabetically, this should come before `WebexInterstitialMeeting`

Reply:
:|